polaris-react/src/components/Columns/Columns.tsx

```diff
@@ -6,22 +6,32 @@ import type {
 
 import {
   getResponsiveProps,
+  getResponsiveValue,
   sanitizeCustomProperties,
+  ResponsiveValue,
 } from '../../utilities/css';
 import type {ResponsiveProp} from '../../utilities/css';
 
 import styles from './Columns.scss';
 
-type Columns = {
-  [Breakpoint in BreakpointsAlias]?: number | string;
-};
+type ColumnWidths = 'oneThird' | 'oneHalf' | 'twoThirds';
```

Comment:
Thoughts on renaming to `ColumnsAlias`?

Reply:
Also, the switch statement allows values that don't match these aliases to fall through. Whereas, this type does not. I believe you want something like:
```tsx
type ColumnAlias = 'oneThird' | 'oneHalf' | 'twoThirds' | (string & {});
```

Reply:
Works for me! In that case we should update the switch statement accordingly.